index.html

```diff
@@ -169,74 +169,73 @@ <h2 id="about-me" class="aboutme_head">About me</h2>
-          <ul>
-            <li>Codekit</li>
-            <li>GitHub</li>
-            <li>Codepen</li>
-            <li>Gitlab</li>
-            <li>Terminal</li>
-          </ul>
-        </li>
-      </ul>
-  </section>
-
-  <section>
-    <footer>
-      <h3 id="contact-info" class="contact_head">I'm always interested in hearing about new projects, so if you'd like
-        to chat please get in touch.</h3>
-      <form novalidate action="https://formspree.io/f/meqdykek" method="post">
-
-        <input type="text" id="full-name" maxlength="30" name="full_name" placeholder="First name" required>
-        <input type="email" id="email" name="user-email" placeholder="Email address"
-          pattern="[a-z0-9._%+-]+@[a-z0-9.-]+\.[a-z]{2,}$" required />
-        <textarea name="message" maxlength="500" placeholder="Enter message here" required></textarea>
-        <div class="errorButton , flex">
-          <button class="form_button" type="submit">Get in touch</button>
-          <small class="collapse"></small>
-        </div>
-      </form>
-      <hr>
-      <ul class="footer_icon">
-        <li><a href="https://twitter.com/JoseADelphin"><img src="assets/images/social_media/twitter.svg"
-              alt="Twitter icon"></a></li>
-        <li><a href="https://www.linkedin.com/in/josedelphin/"><img src="assets/images/social_media/linkedin.svg"
-              alt="Linkedin icon"></a></li>
-        <li><a href="https://github.com/adelphinsucasa"><img src="assets/images/social_media/github.svg"
-              alt="Github icon"></a>
-        </li>
-      </ul>
-      <h4>dalejandro.005@gmail.com</h4>
-    </footer>
+      </div>
+  </div>
+  <div>
+    <ul class="flex_container_secondary">
+      <li class="nested_flexbox">
+        <h3>Languages</h3>
+        <img class="center" src="assets/images/icon-languages.svg" alt="Languages icon">
+        <ul>
+          <li>JavaScript</li>
+          <li>Ruby</li>
+          <li>HTML</li>
+          <li>CSS</li>
+        </ul>
+      </li>
+      <li class="nested_flexbox">
+        <h3>Frameworks</h3>
+        <img class="center ordering" src="assets/images/icon-frameworks.svg" alt="Frameworks icon">
+        <ul>
+          <li>Bootstrap</li>
+          <li>Ruby on rails</li>
+          <li>RSpec</li>
+          <li>Capybara</li>
+          <li>Selenium</li>
+        </ul>
+      </li>
+      <li class="nested_flexbox">
+        <h3>Skills</h3>
+        <img class="center" src="assets/images/icon-skills.svg" alt="Skills icon">
+        <ul>
+          <li>Codekit</li>
+          <li>GitHub</li>
+          <li>Codepen</li>
+          <li>Gitlab</li>
+          <li>Terminal</li>
+        </ul>
+      </li>
+    </ul>
   </section>
-  <section class="collapse" id="popup_container"></section>
+  
+  <footer>
+    <h3 id="contact-info" class="contact_head">I'm always interested in hearing about new projects, so if you'd like
+      to chat please get in touch.</h3>
```

Comment:
- Kindly, style your form and button like Figma and use the same background image.👌



| Figma design                                                             | Your design                                         |
| --------------------------------------------------------------------------------| --------------------------------------------------------- |
|<img src="https://user-images.githubusercontent.com/52010789/206790052-4bb13597-3121-4e7e-95a7-9190f3e3f091.png" width="200">|<img src="https://user-images.githubusercontent.com/52010789/206790037-3262cec1-70fa-4336-bcab-4ab51626e141.png" width="200">|


😎️